### DISCUSSION TITLE
Configurazione Home Assistant - Apple Homekit su due reti diverse

### DISCUSSION THREAD

**Discussion by mbonamoneta** [indomus/forum#2118] [Networking]
Buongiorno. Ho in mente un progetto, ma non so se è fattibile.
Lo scopo è quello di domotizzare un altro locale (Garage) che però non ha la stessa rete di casa. Si connette ad internet tramite un modem 4G. Principalmente mi servirà poter integrare tutto su Home Assistant, per poter poi utilizzare tutto su Apple Homekit. Il nuovo locale avrà un interruttore Sonoff 4CH, una cam di sorveglianza e un sensore apertura porte. Vorrei poter integrare tutto nel sistema Home Assistant con un unico server che terrò in casa e che quindi sarà usato per comandare tutto, casa e garage. È possibile realizzare questo?
Vorrei cercare di utilizzare un unico server HA per poter poi utilizzare un unico hub domestico per HomeKit.
Vi ringrazio in anticipo per i consigli!

**Comment by Scialla** [comment]
Se sono attuatori che lavorano in cloud, e non in locale, li potrai poi integrare nel tuo HomeAssistant tramite le loro integrazioni.
Se non passano dal cloud devi trovare soluzioni diverse, magari con un rasp o similare in garage per poi creare una VPN, oppure altro HA unendolo a quello in casa

**Reply by mbonamoneta** [reply]
Perfetto grazie! A breve mi arriveranno i componenti e inizierò a fare delle prove. Buona giornata!!
